### ISSUE TITLE
Free/gratis culture and entertainment?

### ISSUE THREAD

**Issue by forteller** [flore2003/staythefuckhome#27]
Hi. Great site!

I've been thinking of making a list of (places to get) free* culture and entertainment, and when I saw this website I thought maybe the list should live here?

As @rednaks says in #24 we need to consider peoples mental health.

What do you think?

Here's some of the things I've been thinking of:

Repositories of free culture:
- Librivox (can be browsed through the Audiobooks.com and other apps)
- Gutenberg Project
- Itch.io
- http://topwebfiction.com/
- http://www.openculture.com/
- https://craphound.com/ (books by Cory Doctorow)
- https://www.pnparcade.com/collections/free-games

Specific recommendations:
- Moby Dick on Librivox (one of the best read books there)
- Everything is a remix (great video series on copyright etc)
- Worm audiobook
- Mindustry
- Free Culture by Lawrence Lessig
- Kurtzgesagt
- https://www.handlewithjoy.eu/home/mischief/ (print and play game inspired by Harry Potter)

I know everyone easily can find free entertainment everywhere online, but it might be worth pointing out some great stuff still? 

* as in beer and/or as in freedom, doesn't necessarily be in both at the same time

**Comment by borfast** [comment]
Even more so than in #24, I think this deviates a little too much from the website's focus, which is, I believe, to convince people to stay home. Sure, it would be nice to have some suggestions of stuff people can do but convincing people to stay home is so, so much more important than anything else right now, that I believe it's not in our (humans) interest to deviate in the slightest from that message.

**Comment by forteller** [comment]
It's a million times easier to actually stay home if you have something to do at home is my point. I don't want it on the front page, but a subpage linked to from the front. But just my 2 cents :)

**Comment by borfast** [comment]
Fair enough. Having separate pages with more information might be a good idea.
